recipes/opensmile/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "opensmile" %}
+{% set version = "2.3.0" %}
+{% set sha256 = "41adf2886089b76c000b74a479ac32ed983fc09bd81d1e9df4b0f4999ee71c7f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://audeering.com/download/1318/
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: sh buildStandalone.sh -p {{PREFIX}}
```

Comment:
Could you please add ` || { cat config.log; exit 1; }` temporarily? Would like to get the error information from configure as it is unclear why it is failing?